src/Infrastructure/LeanCode.SmsSender/SmsApiClient.cs

```diff
@@ -63,7 +63,7 @@ public async Task SendAsync(string message, string phoneNumber)
                 parameters["fast"] = "1";
             }
 
-            using (var body = new FormUrlEncodedContent(parameters))
+            using (var body = new FormUrlEncodedContent(parameters!))
```

Comment:
Why `!` there? The object is clearly not null, does the compiler fail to infer that? :thinking:

Reply:
_Argument of type 'Dictionary<string, string>' cannot be used for parameter 'nameValueCollection' of type 'IEnumerable<KeyValuePair<string?, string?>>' in 'FormUrlEncodedContent.FormUrlEncodedContent(IEnumerable<KeyValuePair<string?, string?>> nameValueCollection)' due to differences in the nullability of reference types._

Reply:
Whoa, okay

Reply:
I would say - let's change line 45 to match the `FormUrlEncodedContent` signature.